frontend/src/scenes/session-recordings/player/eventsListLogic.ts

```diff
@@ -6,6 +6,7 @@ import { colonDelimitedDuration } from 'lib/utils'
 import { CellMeasurerCache } from 'react-virtualized/dist/commonjs/CellMeasurer'
 
 export const eventsListLogic = kea<eventsListLogicType>({
+    path: ['scenes', 'session-recordings', 'player', 'eventsListLogic'],
```

Comment:
Hm not quite sure what the context is here. Was this auto-generated by Kea?

Reply:
Yep! Kea-typegen added this change. It's related to the change to esbuild.